### DISCUSSION TITLE
Strapi/Appwrite/Feathersjs Quasar Starters

### DISCUSSION THREAD

**Discussion by dkebler** [quasarframework/quasar#9744] [Ideas / Proposals]
https://github.com/strapi/strapi-starter-nuxt-blog.   Anyone made an equivalent to this nuxt starter using quasar (2.0 maybe) 
or how about with appwrite instead of strapi.  https://github.com/appwrite/appwrite.   Also https://feathersjs.com/

Seems quasar should have these starter examples out there to "compete" with nuxt  and other vue frameworks.   

I personally have a full blown quasar 1.x app using featherjs (uses https://vuex.feathersjs.com/).

Having a basic example integrated with some backend like these makes for easier evaluation/adoption cause, well, every frontend needs some kind of backend to be functional.

**Comment by ontwikkelfabriek** [comment]
@dkebler 

> Having a basic example integrated with some backend like these makes for easier evaluation/adoption cause, well, every frontend needs some kind of backend to be functional.

Agreed. Right now there are no starter-kits with backend integration referenced from the main Quasar website.  For Qv1 there are some to be found at:
https://github.com/quasarframework/quasar-awesome
https://forum.quasar-framework.org/category/9/starter-kits

There are more out there , but you have to find them yourself, like:
https://github.com/alexandesigner/quasar-meteor
https://github.com/quasarframework/quasar-template-meteor
https://github.com/pryspry/quasar-firebase-auth-starter-kit

**Reply by mehmetacoskun** [reply]
Can anybody convert this Vue Plugin https://github.com/gravitybv/vue-strapi/tree/master/src/plugins to Quasar boot?
